spec/redis_spec.cr

```diff
@@ -376,4 +374,20 @@ describe Redis::Client do
       end
     end
   end
+
+  describe "current" do
+    it "allows setting current once and response to Redis.current" do
+      (Redis.current).should eq(nil)
```

Comment:
I'm thinking we can then default this value to `Redis::Client.new`, since this defaults to localhost. That way we have a strict `Redis::Client` return type. What do you think?

Something like this. Still allows setter for a custom client but at minimum returns the default.

```crystal
def self.current=(client : Redis::Client)
  @@current ||= client
end

def self.current : Redis::Client
  @@current || Redis::Client.new
end
```